attendance.php

```diff
@@ -55,6 +55,30 @@
 		PRIMARY KEY(id)
 	) DEFAULT CHARACTER SET=utf8mb4;
 
+	USE braven_attendance;
+	ALTER TABLE attendance_people
```

Comment:
and what i had in mind for this was it wouldn't be alter, but just the create things. I basically use it to at-a-glance see what is supposed to be there and having to scroll up and down to see create vs alter defeats that.

but i can make that edit myself too